activitysim/core/workflow/state.py

```diff
@@ -478,15 +478,12 @@ def load_settings(self) -> State:
             include_stack=False,
         )
 
-        # the settings can redefine the cache directories.
-        cache_dir = raw_settings.pop("cache_dir", None)
```

Comment:
We used "pop" here, this takes this setting off the `raw_settings`.  The revision leaves the setting in place... should we be popping all these values?

I think we need to write a test [or tests] to make sure this is solving the problem well.